### ISSUE TITLE
Include screenshots

### ISSUE THREAD

**Issue by atpalmer** [MicrosoftDocs/azure-stack-docs#7]
I don't follow where to click.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 46d0432d-8bdf-ae6a-000c-4da12913b759
* Version Independent ID: e14583a3-c977-106e-0211-a516734e3965
* Content: [Deploy templates using the portal in Azure Stack](https://docs.microsoft.com/en-us/azure/azure-stack/user/azure-stack-deploy-template-portal)
* Content Source: [articles/azure-stack/user/azure-stack-deploy-template-portal.md](https://github.com/MicrosoftDocs/azure-stack-docs/blob/master/azure-stack/user/azure-stack-deploy-template-portal.md)
* Service: **azure-stack**
* GitHub Login: @sethmanheim
* Microsoft Alias: **sethm**

**Comment by sethmanheim** [comment]
@atpalmer Hi and thanks for reaching out. Can you tell us which part of the procedure you're having an issue with? If you go to the portal you should see the specified options on the left side of the screen. It's a short procedure so we decided not to clutter it with screen shots.

**Comment by atpalmer** [comment]
@sethmanheim It would be even less cluttered if you left off the text description of the steps. Are you trying to make your documentation uncluttered, or are you trying to make it helpful?

**Comment by sethmanheim** [comment]
@atpalmer Thank you for your very polite feedback -- we will consider adding more screenshots to make this topic more helpful.

**Comment by atpalmer** [comment]
@sethmanheim I wasn't being polite, obviously, but I am pretty annoyed with how you answered. This is UI documentation, and there are a million places to click in the Azure UI. Adding screenshots seems like a very small and very appropriate ask.

**Comment by sethmanheim** [comment]
I have added screen shots for each of the steps, and this should be published live by 10 AM tomorrow (May 7), US Pacific Time. Thank you again for the contribution. #please-close

**Comment by atpalmer** [comment]
@sethmanheim Thanks. This does look a lot better, and it is enlightening. Nobody I work with actually has the "Custom" option appear.

We all see essentially this:

![image](https://user-images.githubusercontent.com/26459917/57319738-e5a0d580-70c2-11e9-8b09-a1e53f5aec4d.png)

(The "See all" option also does not take me to a page with a "Custom" option, and searching for "Custom" on the following page also does not return a relevant result.)

Please always take into consideration differences different users encounter, and consider making further improvements with that in mind.

The reason I was annoyed earlier is that I don't think you should be resisting making your documentation better. In my case, this page was essentially useless. I believe it is in your interest to take feedback and improve rather than push back and suggest there is something wrong with me for not figuring something out.

I actually still don't know why my UI looks different, but I did learn that the UI can display different options for different users, so your updates were helpful in the sense that I now know why I can't follow these instructions.

Thank you again for the updates.

**Comment by sethmanheim** [comment]
@atpalmer My last comment on this subject: I don't see any place in my earlier comment where I suggested there was something wrong with you. I thought I was expressing my professional opinion. I actually do not agree that this makes the topic better -- there are far too many screen shots for such a short procedure (with many opportunities for those shots to go outdated as the portal UI changes/updates). If this short procedure is too hard to follow without screen shots, then we need to improve the portal UI (and I have passed this feedback to the engineering team). 

That said, it's good to hear this kind of feedback and to know that more screenshots are helpful. It helps in future doc planning. However, due to frequent updates, it is completely expected that what you see in the screen shots may not be exactly what you see in your portal UI.

Thank you again for reaching out and I hope you continue to do so.

**Comment by atpalmer** [comment]
@sethmanheim But, as you can see, a screenshot you posted *today*, and a screenshot I posted *today* look different. So, that's not an out-of-date issue. It's an issue with the documentation incompletely accounting for users who see my UI. (I am not alone.)

Moreover, it's actually for this same reason that the screenshots **continue to be valuable** if they go out of date: it's possible to _see_ that the instructions are referring to an outdated UI, and the user can comfortably disregard them rather than being confused. It clearly does not help anyone to try to follow directions for an outdated UI. I do hope that you reconsider your opinion and the implications of your opinion on confused, frustrated, annoyed users.

Thank you again for today's updates.

**Comment by ianc911** [comment]
Ha! I too found this page because I'm trying to figure out how to deploy from a template. I too don't see the 'Custom' choice in the portal, but only see what's in atpalmer's screenshot from 7 May above. Not very helpful..
